### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by fuweifuvtoo** [zhiyuanyou/UniAD#11]
thanks for your excellent work
I still have a few questions：
1. Whether the learning rate needs to be modified when switching to different number of Gpus;
2. If I want to use UniAD to train a single class, in order to verify Table 1 and table 2 in the paper, do I only need to restrict the input data to a single class? I have done this, but I can not achieve the result in some class, such as screw, I only got 84.6 detection AUROC metric;

**Comment by zhiyuanyou** [comment]
Hi, ~ 
1. We do not modify the learning rate with different GPUs, thus we set a big epoch number for enough training.
2.  Actually, after we finished our code refactoring, we only test it with the unified setting, which is our setting and focus in the paper. We will check our initial code and setting in the future. Sorry not now, since I am very busy for my graduation currently.

**Comment by fuweifuvtoo** [comment]
thanks for your reply and good luck with your graduation ~
